test/integration/integration_test.go

```diff
@@ -169,10 +170,10 @@ func FeatureContext(s *godog.Suite) {
 		getRoutingUrlAndVerifyHTTPResponse)
 
 	// steps for verifying config file content
-	s.Step(`JSON config file "([^"]*)" (contains|does not contain) key "(.*)" with value "(.*)"$`,
-		configContains)
-	s.Step(`JSON config file "([^"]*)" (contains|does not contain) key "(.*)"(.*)$`,
-		configContains)
+	s.Step(`^JSON config file "([^"]*)" (contains|does not contain) key "(.*)" with value "(.*)"$`,
+		jsonConfigContains)
+	s.Step(`^JSON config file "([^"]*)" (has|does not have) key "(.*)" with value assigned$`,
```

Comment:
@gbraad Actually I had it without `with value assigned` and changed it just before the PR, if your suggestion is to remove the `with value assigned` I am +1 and will change it on next update.

Thanks for review, Gerard.